kafka_consumer/tests/conftest.py

```diff
@@ -23,11 +23,8 @@
 
 E2E_METADATA = {
     'custom_hosts': [('kafka1', '127.0.0.1'), ('kafka2', '127.0.0.1')],
-    'start_commands': [
-        'apt-get update',
-        'apt-get install -y build-essential',
-    ],
-}
+    'docker_volumes': ['{}/start_commands.sh:/tmp/start_commands.sh'.format(HERE)],
```

Comment:
This is only temporary for now. Once https://github.com/DataDog/integrations-core/pull/14022 is merged in we can remove this part.